docs/design/job-resource-reservation-design.md

```diff
@@ -1,148 +0,0 @@
-# Volcano Resource Reservation For Target Jobs
```

Comment:
Maybe we should have a deprecated directory and put this design document in it instead of deleting it directly. This way users can also refer to the previous documents. @Monokaix WDYT

Reply:
> Maybe we should have a deprecated directory and put this design document in it instead of deleting it directly. This way users can also refer to the previous documents. @Monokaix WDYT

If users want to view old documents, they can switch to the old branch

Reply:
We can mark it as deprecated and delete it in next version: )

Reply:
@Monokaix 
how to mark it as deprecated? write in title? create a deprecated folder?

Reply:
I think we can just edit the doc directly: ), and whether it's sovled by sla-plugin should also be checked.